<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>domClass</class>
 <widget class="QMainWindow" name="domClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>990</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dom</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="tabsMovable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>算子</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>图像预处理</string>
     </property>
     <addaction name="image_segmentation_action"/>
     <addaction name="image_stitch_action"/>
     <addaction name="action_4"/>
     <addaction name="action_5"/>
    </widget>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>特征处理</string>
     </property>
     <addaction name="action_7"/>
     <addaction name="action_8"/>
     <addaction name="action_9"/>
     <addaction name="action_10"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="menu_3"/>
    <addaction name="action_11"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="input_window_action"/>
    <addaction name="operator_window_action"/>
    <addaction name="output_window_action"/>
    <addaction name="variate_window_action"/>
    <addaction name="action_test"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="image_segmentation_action">
   <property name="text">
    <string>图像分割</string>
   </property>
  </action>
  <action name="image_stitch_action">
   <property name="text">
    <string>图像拼接</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>去边缘毛丝</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>去内部孔洞</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>边缘检测</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>形状筛选</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>矩形筛选</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>像素阈值筛选</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>相机接口</string>
   </property>
  </action>
  <action name="operator_window_action">
   <property name="text">
    <string>打开算子窗口</string>
   </property>
  </action>
  <action name="input_window_action">
   <property name="text">
    <string>打开输入窗口</string>
   </property>
  </action>
  <action name="output_window_action">
   <property name="text">
    <string>打开输出窗口</string>
   </property>
  </action>
  <action name="input_code_action">
   <property name="text">
    <string>代码窗口</string>
   </property>
  </action>
  <action name="variate_window_action">
   <property name="text">
    <string>打开变量窗口</string>
   </property>
  </action>
  <action name="action_test">
   <property name="text">
    <string>传值test</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dom.qrc"/>
 </resources>
 <connections/>
</ui>
